bbcdll/bbc_tcp_server.py

```
@@ -334,10 +364,14 @@ def load_config(filename):
             SS["snapshotDevice"] = page.SS.get("snapshotDevice", None)
             SS["operateDevice"] = page.SS.get("operateDevice", None)
             page.SS = SS
-            page.reset()
+            # 线程安全：page.reset() 可能触发 UI 重绘
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)